### DISCUSSION TITLE
Frame not rendering specify path

### DISCUSSION THREAD

**Discussion by 5675sp** [h2oai/wave#867] [General]
Hello, 

I am not sure if this a bug, but the path I've specified in the frame above is not being rendered. Can someone please advise? Thank you. 

```
from h2o_wave import main, app, Q, ui

async def setup_page(q: Q):

    q.page['driverless_ai'] = ui.form_card(
        box='1 1 -1 -1',
        items=[
            ui.frame(path='https://twitter.com', height='800px' ),
            ui.link(label='External link', path='https://twitter.com')
        ]
    )
	

  
    
@app('/tutorial')
async def serve(q: Q):


    if not q.client.initialized:
        q.client.initialized = True


    await setup_page(q)


    await q.page.save()
```

**Comment by vopani** [comment]
Many websites (including Twitter) don't allow direct embedding in an iframe (GitHub is another example). Specifically for Twitter, they have some HTML publishing options: https://publish.twitter.com/#

**Reply by 5675sp** [reply]
I see, thank you @vopani
